agent/api/ecsclient/client_test.go

```diff
@@ -269,6 +292,20 @@ func TestRegisterContainerInstance(t *testing.T) {
 	}
 }
 
+func TestRegisterInstanceBadAttribute(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+	err := awserr.New("ClientException", "Invalid attribute given", errors.New("Invalid attribute given"))
```

Comment:
Sorry, this code shouldn't have been committed. It was going to test something that never actually got implemented.